actions/command.ts

```diff
@@ -140,6 +141,15 @@ export function executeCommand(message: string, args: CommandArgs): ActionFunc {
 
             dispatch(openModal({modalId: ModalIdentifiers.PLUGIN_MARKETPLACE, dialogType: MarketplaceModal}));
             return {data: true};
+        case '/worktemplate': {
```

Comment:
Is `Work Template` the final name? It seems we were converging with `Templates`, wondering if this should be then just `/templates`?

Reply:
I did wonder, but `/template(s)` is so generic 😅

Reply:
I'd prefer `/templates` as well, but maybe more a call for UX / PO.